### DISCUSSION TITLE
Inline tools to block

### DISCUSSION THREAD

**Discussion by ar53n** [codex-team/editor.js#1908] [Q&A]
Add children for blocks. And can add opportunity convert inline tools in blocks?

```
<p>123<b>4</b>56<p>
now its look like a 
{
    type: "paragraph"
    data: {
        "text": "123<b>4</b>56"
    }
}
but i want convert to 

{
            "type" : "paragraph",
            "data" : {
                "text" : "123",
                "childrens": [
                   {
                     type:"bold",
                     text: "4"
                   }
                ]
            }
},
```

**Comment by neSpecc** [comment]
Could you please explain your use case. For what purposes you want to use this structure? How you will render it?

**Comment by ar53n** [comment]
> Could you please explain your use case. For what purposes you want to use this structure? How you will render it?

I have component <Bold>{text}</Bold> with css modules classes and i want use him. Or maybe we can add support css modules?

**Comment by neSpecc** [comment]
For now, editorjs stores inline fragments inside the block’s data as HTML structures. The reason is simplifying render process. We are planning to add an option on how to store these fragments in output data: as inline-html, as markdown or as a JSON object
